client/state/jetpack-connect/schema.js

```diff
@@ -2,7 +2,7 @@ export const jetpackConnectSessionsSchema = {
 	type: 'object',
 	additionalProperties: false,
 	patternProperties: {
-		'^[a-z0-9\.\-\:\/]+$': {
+		'^[a-z0-9\.\:\/\-]+$': {
```

Comment:
hmm, you have a point. The urls are not coming from an endpoint, but entered by users, but to be honest, we don't gain a lot by checking the pattern